packages/medusa/src/api/routes/admin/price-lists/delete-product-prices.ts

```diff
@@ -0,0 +1,48 @@
+import PriceListService from "../../../../services/price-list"
+
+/**
+ * @oas [delete] /price-lists/{id}/products/{product_id}
```

Comment:
```suggestion
 * @oas [delete] /price-lists/{id}/products/{product_id}/prices
```

Reply:
This would require an update of the operationId too. Specifically we'd have to include Prices at the end of my suggestion

Reply:
Yes indeed, now that I know the composition i ll do the rest based on the end points ive created including the prices 👌

Reply:
@adrien2p Sounds good!